<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenPlascilabApp</class>
 <widget class="QWidget" name="FenPlascilabApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plascilab</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="barreOnglets">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="ongletHorairesRER">
      <attribute name="title">
       <string>Horaires RER</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWebEngineView" name="pageWeb" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="url" stdset="0">
          <url>
           <string>http://monrer.fr/?s=RIS</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ongletAdministration">
      <attribute name="title">
       <string>Administration</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
       <item>
        <layout class="QVBoxLayout" name="layoutVueAdherents" stretch="1,0">
         <item>
          <widget class="QTableView" name="vueTableAdherents"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutDetailsBeneficiaires" stretch="1,0">
           <item>
            <widget class="QGroupBox" name="gbDetails">
             <property name="title">
              <string>Détails</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="6" column="0">
               <widget class="QLabel" name="labelCP">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;CP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelNom">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="4" column="3">
               <widget class="QLabel" name="affTypeAbn">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLabel" name="affMail">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QLabel" name="labelAbonnement">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Abonnement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="affDesignation">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLabel" name="affTel">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QLabel" name="affVille">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="labelTel">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Tel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="5" column="3">
               <widget class="QLabel" name="affAbonnement">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="affNom">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="labelDesignation">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Désignation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QLabel" name="labelTypeAbn">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Adhésion&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QLabel" name="labelDebutAbn">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Début Abn&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="labelPrenom">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Prénom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="affPrenom">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="labelAdresse">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Adresse&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="6" column="3">
               <widget class="QLabel" name="affDebutAbn">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLabel" name="affAdresse">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="labelVille">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Ville&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="labelMail">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Mail&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="affIDCarte">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="7" column="2">
               <widget class="QLabel" name="labelFinAbn">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Fin Abn&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLabel" name="affCP">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="7" column="3">
               <widget class="QLabel" name="affFinAbn">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="labelIDCarte">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ID Carte&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="labelID">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;N° Adhérent&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="affID">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="gbBeneficiaires">
             <property name="title">
              <string>Bénéficiaires</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTableView" name="vueTableBeneficiaire"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="bAjouterBeneficiaire">
                  <property name="text">
                   <string>Ajouter bénéficiaire</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="bSupprimerBeneficiaire">
                  <property name="text">
                   <string>Supprimer bénéficiaire</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layoutGBAdministration" stretch="0,0,0,0">
         <item>
          <widget class="QGroupBox" name="gbConnexion">
           <property name="title">
            <string>Connexion</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="bConnexion">
              <property name="text">
               <string>Connexion</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bParametres">
              <property name="text">
               <string>Paramètres</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbRecherche">
           <property name="title">
            <string>Recherche</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="lineRecherche">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bRecherche">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Go</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gbBoutonsAdministration">
           <property name="title">
            <string>Administration</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QPushButton" name="bRafraichir">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Rafraîchir la liste</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bAbnNonAJour">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Adhérents non à jours</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bAjouterAdh">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Ajouter adhérents</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bCrediterAdh">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Créditer adhérents</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bEditerAdh">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Editer adhérents</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bSupprimerAdh">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Supprimer adhérents</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ongletPassages">
      <attribute name="title">
       <string>Passages</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="ongletEtatServices">
      <attribute name="title">
       <string>Etat services</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="layoutEtatServices" columnstretch="0,1,0">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="labelEtatServices">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Etat des services&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="affServeurSQL">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelServeurSQL">
           <property name="text">
            <string>Serveur SQL : </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelSeveurRPi">
           <property name="text">
            <string>Serveur Raspberry : </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="affServeurRPi">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="bEteindreRPi">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Eteindre Raspberry Pi</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
      <item>
       <widget class="QFrame" name="fBadgeAfficheur">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="labelBadgePrenom">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Prenom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="affBadgeFinAbn">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelBadgeDebutAbn">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Début abonnement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelBadgeNom">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Nom&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelBadgeID">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ID&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="labelBadgeFinAbn">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Fin abonnement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="affBadgePrenom">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="affBadgeNom">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="affBadgeDebutAbn">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="affBadgeID">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAbonnementValide">
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
